app/src/main/java/edu/uc/hickmadc/customnotifications/MainActivity.kt

```diff
@@ -1,30 +1,40 @@
 package edu.uc.hickmadc.customnotifications
 
-import android.annotation.SuppressLint
 import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
-import android.widget.Button
-import android.widget.ImageButton
-import android.widget.Toast
+import android.widget.ListView
 import androidx.navigation.Navigation
 import androidx.navigation.findNavController
 import androidx.navigation.ui.NavigationUI
 import com.google.android.material.bottomnavigation.BottomNavigationView
 import edu.uc.hickmadc.customnotifications.ui.main.MyListAdapter
-import edu.uc.hickmadc.customnotifications.ui.main.dialogfragment
 
+//import edu.uc.hickmadc.customnotifications.ui.main.MainFragment
 
 class MainActivity : AppCompatActivity() {
 
-    override fun onCreate(savedInstanceState: Bundle?) {
+    //temp data
+    val language = arrayOf<String>("Brush Teeth","Eat Breakfast","Do schoolwork")
+    val description = arrayOf<String>(
+        "Brush teeth for 2 minutes",
+        "Eat so you don't feel like shit",
+        "")
 
+    override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
 
 
+        val lView= findViewById<ListView>(R.id.listView)
+        val myListAdapter = MyListAdapter(this,language,description)
+        lView.adapter = myListAdapter
         if (savedInstanceState == null) {
             setupBottomNavigationBar()
+//            supportFragmentManager.beginTransaction()
+//                .replace(R.id.container, MainFragment.newInstance())
+//                .commitNow()
```

Comment:
Committing commented out code is considered bad practice in version control, because version control offers better alternatives:
1) Use a feature branch for experimental features.
2) Delete what is not needed.  You can always compare with history to see what used to be there.
Commented out code is considered clutter in a version control system.